napari/components/tests/component_tests.py

```diff
@@ -0,0 +1,23 @@
+from napari.components.component import Component
```

Comment:
These tests reference methods and attributes that do not exist on `Component`.

Reply:
Yes, we can get rid of the listener tests, that's obsolete

Reply:
removing this now